### DISCUSSION TITLE
Question regarding template assets

### DISCUSSION THREAD

**Discussion by richard-tate-cf** [Accenture/sfmc-devtools#944] [General]
Just wanted to know if there any know issues with using this tool with html email templates, when trying to update an existing template?

**Comment by JoernBerkefeld** [comment]
just one at the moment: 
- #798


You can check the list itself though - we don't keep hidden secondary list anywhere: https://github.com/Accenture/sfmc-devtools/issues?q=is%3Aopen+label%3Ac%2Fasset+label%3Abug+type%3Aissue+

**Comment by richard-tate-cf** [comment]
Ok because we have a custom html template that we are trying to move from our Development Business unit to our QA  Business unit it contains custom blocks that we have built using this https://www.npmjs.com/package/sfmc-sdk package.
The blocks are being moved over but the html markup is not, even though html file is showing in git and we can see the the file in bitbucket diff. I had noticed that template files where added to .gitignore file, but I removed this line for testing and wanted to know if html can be moved from one business unit to an other using your tool?

**Comment by JoernBerkefeld** [comment]
uhm... sfmc-sdk is a generic wrapper around SFMC's APIs. mcdev is using that under the hood. Not sure I understand what you were doing with it.

how exactly are you doing the "moving"? could you please detail the steps that you were following, including commands you executed?

Copying email templates (asset/message folder) is definitely supported. If it didn't work you either found a bug or did it wrong somehow.

**Comment by JoernBerkefeld** [comment]
Duplicate of #949
